webview/src/plots/components/styles.module.scss

```
@@ -156,17 +155,28 @@ $gap: 20px;
   }
 }
 
-.plotHorizontalResizer,
-.plotVerticalResizer {
-  position: absolute;
-  border: none;
-  background: none;
+.ratioSmaller .plot {
+  aspect-ratio: 2 / 1;
+}
 
-  &::after {
-    content: '';
-    background-color: $fg-color;
-    position: absolute;
-  }
+.ratioSmall .plot {
+  aspect-ratio: 9 / 5;
+}
+
+.ratioRegular .plot {
+  aspect-ratio: 4 / 3;
+}
+
+.ratioSquare .plot {
+  aspect-ratio: 1;
+}
+
+.ratioVerticalNormal .plot {
+  aspect-ratio: 3 /4;
```

I'm assuming we want to keep things consistent 😄 
```suggestion
  aspect-ratio: 3 / 4;
```